app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,46 @@
+class Api::V1::UsersController < ApplicationController
+  before_action :set_user, except: [:index]
+
+  def index
+    # sometimes we need to serve all the users
+    # sometimes we need to serve only the requested "batch" -- the specific ids
+    # that the ember app asked for
+    if params[:ids].nil?
+      # [{a: "user"}, {another: "user"}]
+      # {"users": [{id: 1.....}, {id: 2....}]}
+      render json: {"users" => User.all.as_json}
+    else
+      # params[:ids] - gives us the ids of the users
+      # we need to find for ember
+      render json: {"users" => User.where(:id => params[:ids]).as_json}
+    end
+  end
```

Comment:
I'm not actually sure we need an index for users. Do we need to serve all the users or even a batch of users for the purposes of our app?